scrapy/core/downloader/handlers/aiohttp.py

```diff
@@ -0,0 +1,52 @@
+"""Download handlers for http and https schemes"""
+import logging
+import asyncio
+
+import aiohttp
+from twisted.internet import defer
+
+from scrapy.http import Headers
+from scrapy.responsetypes import responsetypes
+
+logger = logging.getLogger(__name__)
+
+
+class HTTPDownloadHandler(object):
+
+    def __init__(self, settings):
+        self.settings = settings
+
+    def download_request(self, request, spider):
+        return _force_deferred(self._download_request(request, spider))
+
+    def _download_request(self, request, spider):
+        """Return a deferred for the HTTP download"""
+        headers=list((k.decode('latin1'), v.decode('latin1'))
+                     for k, vs in request.headers.items()
+                     for v in vs)
```

Comment:
All non-ascii header values will be sent incorrectly because aiohttp [encodes](https://github.com/KeepSafe/aiohttp/blob/bfea76495c2b3f76e51f9a60bc4a1b7fe161e183/aiohttp/protocol.py#L651) headers to utf8 before sending.